cirq/ops/common_channels.py

```diff
@@ -0,0 +1,188 @@
+        if not isinstance(other, type(self)):
+            return NotImplemented
+        return self._eq_tuple() == other._eq_tuple()
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __repr__(self) -> str:
+        return (
+            '(cirq.AsymmetricDepolarizingChannel(p_x={!r},p_y={!r},p_z={!r}))'
+                .format(self._p_x, self._p_y, self._p_z)
+        )
+
+    def __str__(self) -> str:
+        return ('AsymmetricDepolarizingChannel(p_x={!r},p_y={!r},p_z={!r})'
+                .format(self._p_x, self._p_y, self._p_z))
+
+    def _circuit_diagram_info_(self,
+        args: protocols.CircuitDiagramInfoArgs) -> str:
+        return 'A({!r},{!r},{!r})'.format(self._p_x, self._p_y, self._p_z)
+
+
+def asymmetric_depolarize(
+    p_x: float,
+    p_y: float,
+    p_z: float) -> AsymmetricDepolarizingChannel:
+    """Returns a AsymmetricDepolarizingChannel with given parameter.
+
+    This channel evolves a density matrix via
+        \rho -> (1 -p_x + p_y + p_z) \rho
+                + p_x X \rho X + p_y Y \rho Y + p_z Z \rho Z
+
+    Args:
+        p_x: The probability that a Pauli X error occurs.
+        p_y: The probability that a Pauli Y error occurs.
+        p_z: The probability that a Pauli Z error occurs.
+
+    Raises:
+        ValueError if the args or the sum of the args are not probabilities.
+    """
+    return AsymmetricDepolarizingChannel(p_x, p_y, p_z)
+
+
+class DepolarizingChannel(raw_types.Gate):
+    """A channel that depolarizes a qubit."""
+
+    def __init__(self, p) -> None:
+        """The symmetric depolarizing channel.
+
+        This channel applies one of the three Pauli operators at random,
+        or does nothing. The probability of each of the Pauli operator is
+        equal and is given by p / 3 where p is the arg to this constructor.
+        The given probability must be a valid probability (between 0 and 1).
+
+        This channel evolves a density matrix via
+            \rho -> (1 - p) \rho
+                    + (p / 3) X \rho X + (p / 3) Y \rho Y + (p / 3) Z \rho Z
+
+        This channel can be repeated an integer number of times by raising
+        the channel to a power.
+
+        Args:
+            p: The probability that one of the Pauli gates is applied, each of
+                the Pauli gates being applied with equal probability (p / 3).
+
+        Raises:
+            ValueError if p is not a valid probability.
+        """
+
+        self._p = p
+        self._delegate = AsymmetricDepolarizingChannel(p / 3, p / 3, p /3)
+
+    def _channel_(self) -> Iterable[np.ndarray]:
+        return self._delegate._channel_()
+
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return NotImplemented
+        return self._p == other._p
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __repr__(self) -> str:
+        return '(cirq.DepolarizingChannel(p={!r}))'.format(self._p)
```

Comment:
Unnecessary parens on repr. Consider using `cirq.depolarize(p)` instead, since that's the language users will use.